sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTypeWideningSuite.scala

```diff
@@ -166,11 +166,17 @@ class ParquetTypeWideningSuite
       (Seq("1", "2", Int.MinValue.toString), LongType, IntegerType),
       (Seq("1.23", "10.34"), DoubleType, FloatType),
       (Seq("1.23", "10.34"), FloatType, LongType),
-      (Seq("1.23", "10.34"), LongType, DateType),
-      (Seq("1.23", "10.34"), IntegerType, TimestampType),
-      (Seq("1.23", "10.34"), IntegerType, TimestampNTZType),
       (Seq("2020-01-01", "2020-01-02", "1312-02-27"), DateType, TimestampType)
-    )
+    ) ++ {
+      if (SQLConf.get.ansiEnabled) Nil
```

Comment:
It seems that only the test cases used the input test data that would fail to type conversion when ANSI MODE is set to true when constructing test cases that `would throw errors when read`. In this PR, To quickly fix , I have skipped these three test inputs

@cloud-fan @zhengruifeng FYI